prbt_hardware_support/test/unittests/unittest_operation_mode_setup_executor.cpp

```diff
@@ -209,48 +208,48 @@ TEST_F(OperationModeSetupExecutorTest, testUpdateOperationModeUnknownMode)
   op_mode.time_stamp = ros::Time::now();
   op_mode.value = OperationModes::UNKNOWN;
 
-  EXPECT_CALL(*this, setSpeedLimit(Le(0.0))).WillOnce(Return(true));
+  EXPECT_CALL(*this, monitorCartesianSpeed(_)).Times(0);
 
   executor_->updateOperationMode(op_mode);
 }
 
-class SetSpeedLimitServiceMock
+class MonitorCartesianSpeedServiceMock
 {
 public:
-  MOCK_METHOD1(call, bool(SetSpeedLimit& srv));
+  MOCK_METHOD1(call, bool(std_srvs::SetBool& srv));
   MOCK_METHOD0(getService, std::string());
 };
 
-MATCHER_P(IsCorrectSpeedLimitSet, speed_limit, "") { return arg.request.speed_limit == speed_limit; }
+MATCHER_P(IsMonitorCartesianSpeedCorrect, active_flag, "") { return arg.request.data == active_flag; }
```

Comment:
Function name is not really fitting. I suggest something like: `isSpeedMonitoringSettingCorrect`.

Reply:
Done.